### ISSUE TITLE
wagtail_urls?

### ISSUE THREAD

**Issue by qyb** [coderedcorp/wagtail-cache#15]
https://docs.coderedcorp.com/wagtail-cache/stable/getting_started/install.html#caching-wagtail-pages-only:

 ... url(r'', include(wagtail_urls)),

But in the newest version, it seems that "re_path(r'', include(codered_urls))," replace the older define?

**Comment by vsalvino** [comment]
Good catch, feel free to make a pull request updating the documentation. For reference, that specific code snippet should match the default `urls.py` file generated by `wagtail start`.
